lib/vpc-infrastructure-stack.ts

```diff
@@ -4,12 +4,48 @@
 
 import * as cdk from 'aws-cdk-lib';
 import { Construct } from 'constructs';
-import { RemovalPolicy } from 'aws-cdk-lib';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as config from 'config';
+import { IpAddresses } from 'aws-cdk-lib/aws-ec2';
 
 export class VpcInfrastructureStack extends cdk.Stack {
+    nelsonVPC: ec2.Vpc;
     constructor(scope: Construct, id: string, props?: cdk.StackProps) {
         super(scope, id, props);
 
-        //TODO: Create a VPC here
```

Comment:
Remove the TODO comment. I believe the VPC code is added so it no longer is a TODO